site/_posts/2017-03-24-pontificatory-requestiveous.md

```diff
@@ -0,0 +1,111 @@
+---
+title: "Last Week in ManageIQ: Pontificatory Requestiveous - PR"
+author: syncrou
+date: 2017-03-24 13:34:37 -04:00
```

Comment:
Not sure, but do we want to update the date to reflect published date (presumably) of tomorrow? Wondering if I could update the generator to make this `today.tuesday? ? today : next_tuesday` :thinking: